### DISCUSSION TITLE
Thonny says file not found even when the file is there.

### DISCUSSION THREAD

**Discussion by AppalachianWarrior** [thonny/thonny#3234] [General]
I am an Arduino/C++ guy and I am sorry if I use a different 'lingo' to explain the issue.

I hit F5 to run this file named TestBed1.py. The Thonny IDE is saying that it can't find the ble_advertising.py header file.

In the image you can see that the ble_advertising.py file is indeed in the same folder as TestBed1.py

Why is the Thonny IDE saying this when the required file is indeed present in the same folder as the file that I am trying to execute?

![twe](https://github.com/thonny/thonny/assets/72459209/f92dfd74-a7f2-42b0-87e4-ccd0320c9e96)

**Comment by Hermann12** [comment]
You have to show the import statement, too. Can you edit your question, please.

**Comment by aivarannamaa** [comment]
MicroPython imports modules from the root of the filesystem by default. Your file is ble_advertising is in a subfolder.
